includes/import-products.php

```diff
@@ -29,9 +29,17 @@ function dw_import_products() {
 	$filteredCourses = $courseFilter->filter_courses($dw_options['ccv_code']);
 
 	foreach($filteredCourses as $dationProduct) {
-		if(dw_get_product_by_sku($dationProduct['id']) === null) {
+		$woocommerceProduct =  dw_get_product_by_sku($dationProduct['id']);
```

Comment:
We leave it up to the user to make sure that all fields are there? Because in Dation all fields are optional for a Location, so this could produce `",,,,"`.